### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by bioSandMan**
basic tests are working.

### CODE REVIEW

File: test_suite/testSD.py [hunk 1]
```diff
@@ -14,40 +19,68 @@ def findFastqFiles(directory, pattern):
...
+        self.assertEqual(find_fastq_files('fastqFiles', '*.fastq'),
+                         ['fastqFiles/expected_R1.fastq',
+                          'fastqFiles/testCase_1X_R1.fastq',
+                          'fastqFiles/testCase_1X_R2.fastq'],
+                         "Unable to find test files")
+
+    def test_tab_output(self):
+        """Tests for 'Reads_Read' in the output"""
+        self.assertIn("Reads_Read\tReads_Written\tReads_Discarded\t"
+                      "Singletons\tDoubles\tThree_Plus\t"
```
Comment (dstreett) [inline test_suite/testSD.py:62]: Hey @bioSandman. It looks great but I just changed up the format to add "Reads_Per_Second" so it is failing "Unexpected tab structure". It is on latest develop branch. Thank you!

**Comment by dstreett** [comment]
Hey, @bioSandMan . One more question - when I merged it into develop, I keep getting an error "Unable to find test files". I pushed it up to test_develop (with proposed indent change as well).
Any suggestions on why this is happening?

Thank you for your help!

**Comment by bioSandMan** [comment]
Did you figure this one out? If it's complaining about the test files then
it's unable to find one of the fastq datasets. The test is looking for
specific files, so either the files don't exist or it's finding extra fastq
files that it didn't expect. Check to make sure your .gitignore isn't
listing fastq files or add them explicitly.

**Comment by bioSandMan** [comment]
Found it. It was on my end. I decided that we didn't need to test the file
compare since I was testing specific expected entries in the output. Also,
my merge isn't pulling the changes in the output for Super Deduper.

**Comment by dstreett** [comment]
Fetch origin/develop and then merge develop into your testing branch and you should see it.

I pushed a branch (maybe testing_develop or something like that) yesterday which should have done this exact thing if you want to look at it.

Let me know if you have any questions.

**Comment by bioSandMan** [comment]
I merged test_develop into testing and everything seems to be working. I pushed the changes back to the testing branch.

**Pull request merged**